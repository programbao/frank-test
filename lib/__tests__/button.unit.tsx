import React from "react";
import renderer from "react-test-renderer";
import Button from "../button";
describe("我的第一个测试用例", () => {
  it("是一个div", () => {
    const json = renderer.create(<Button />).toJSON();
    expect(json).toMatchSnapshot();
  });
});
