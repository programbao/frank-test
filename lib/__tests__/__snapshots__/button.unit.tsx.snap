// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`我的第一个测试用例 是一个div 1`] = `
<div>
  按钮
</div>
`;
